estate/models/estate_property.py

```diff
@@ -0,0 +1,20 @@
+from odoo import fields, models
+
+
+class EstateProperty(models.Model):
+    _name = "estate_property"
+    _description = "Estate Property"
+    name = fields.Char(required=True)
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date()
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float()
+    bedrooms = fields.Integer()
+    living_area = fields.Integer()
+    facades = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
+    garden_area = fields.Integer()
+    garden_orientation = fields.Selection(string="Orientation", selection=[("North", "North"), ("South", "South"), ("East", "East"), ("West", "West")])
```

Comment:
I would prefer to right the selection options that way to make it more clearer Also the tuple first value represent the technical name so it should be in snake_case 🐍 
```suggestion
    garden_orientation = fields.Selection([
        ("north", "North"),
        ("south", "South"),
        ("east", "East"),
        ("west", "West")
    ], string="Orientation")
```